examples/ordered.rs

```diff
@@ -3,21 +3,11 @@ use cc_traits::{Back, Collection, PushBack};
 use std::collections::VecDeque;
 
 /// Ordered stack.
+#[derive(Default)]
 pub struct Ordered<S> {
 	inner: S,
 }
 
-impl<S> Ordered<S> {
```

Comment:
I agree a `Default` impl is necessary, but please also keep the `new` constructor, it is pretty standard.

Reply:
Fixed